qa/L0_trt_error_propagation/trt_error_propagation_test.py

```diff
@@ -40,12 +40,23 @@ def test_invalid_trt_model(self):
         with self.assertRaises(InferenceServerException) as cm:
             self.__triton.load_model("invalid_plan_file")
         err_msg = str(cm.exception)
-        self.assertIn("Internal: unable to create TensorRT engine", err_msg,
-                      "Caught an unexpected exception")
+        # "[err_msg]" -> "[triton_err_msg]: [backend_err_msg]: [trt_err_msg]"
+        backend_err_msg = "Internal: unable to create TensorRT engine: "
```

Comment:
Can you use prefix terms like `expected_` and `got_` to make logic more readable?

Reply:
Yes. Update commit: [Clarify source of error messages](https://github.com/triton-inference-server/server/pull/5719/commits/f4cbf55bb578b2529c7460aedae0c8cc2d1d414d)